knimin/lib/data_access.py

```diff
@@ -2443,35 +2449,28 @@ def _study_is_unique(self, qiita_study_id=None, title=None, skip_id=None):
         ValueError
             If neither value is given, or either value already exists
         """
-        tags = ['Qiita study ID', 'Title']
-        cols = [tag.lower().replace(' ', '_') for tag in tags]
+        cols = ['qiita_study_id', 'title']
         vals = [vars()[col] for col in cols]
```

Comment:
Any specific reason why you keep the call to `vars()`? That is one of the changes that I proposed in my version and the reason behind is that the current way of doing it becomes not that readable.

This version:
```
vals = [qiita_study_id, title]
```
is more readable and easy to maintain... There is no reason for making this call so "meta"